src/components/pagination/pagination.js

```diff
@@ -0,0 +1,143 @@
+import '@brightspace-ui/core/components/button/button-icon.js';
+import '@brightspace-ui/core/components/inputs/input-number.js';
+import '@brightspace-ui/core/components/inputs/input-text.js';
+import { css, html, LitElement } from 'lit';
+import { LocalizeLabsElement } from '../localize-labs-element.js';
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+
+class Pagination extends RtlMixin(LocalizeLabsElement(LitElement)) {
+
+	static get properties() {
+		return {
+			pageNumber: { type: Number, attribute: 'page-number', reflect: true },
+			maxPageNumber: { type: Number, attribute: 'max-page-number', reflect: true },
+			showItemCountSelect : { type: Boolean, attribute: 'show-item-count-select', reflect: true },
+			itemCountOptions : { type: Array, attribute: 'item-count-options' },
+			selectedCountOption : { type: Number, attribute: 'selected-count-option', reflect: true },
+		};
+	}
+
+	static get styles() {
+		return [selectStyles, css`
+			:host {
```

Comment:
A couple things were removed here. `align-content` and `flex-direction`. Are these not needed here?

Reply:
In these cases, they were just assigning the default value, and therefore could get removed since they weren't providing any benefit